markdown/solace-masterclass/solace-masterclass.md

```diff
@@ -752,7 +752,31 @@ Confirmed_ and corresponding _Account Suspended_ events.
 
 #### How to run ?
 
-- Navigate to the directory: **/home/ubuntu/GitHub/solace-masterclass-code/banking-domain/account-management**
+##### **Part 1 : Broker Configuration**
+- Click on the **Designer** button and enter the Designer screen where you should be able to see the newly imported Application Domain.
+![designer-app-domain-imported.png](img/banking-domain-usecase/designer-app-domain-imported.png)
+- Click on the **Acme Bank** application domain, where you can visualise the events, applications and their interactions as below :
+  ![formatted-app-domain.png](img/banking-domain-usecase/formatted-app-domain.png)
+> aside positive Do note that the first time visualization of the events and their interactions might differ from what you see in the screenshot. \
+> You can reorder the objects as per your preference
+- Click on the **Account Management**, and then on the **Open Application** which pops up as below :
+  ![edit-account-management-1.png](img/banking-domain-usecase/edit-account-management-1.png)
+- The **Account Management** application screen displays you the various events that are published and subscribed by this version of the application. It also describes the various environments this application is deployed on :
+  ![editing-acc-management-1.png](img/banking-domain-usecase/editing-acc-management-1.png)
+- Click on the **Runtime** tab at the top of the application, and you can see the various environments, brokers and the status of the application configuration deployment as below :
+  ![acc-mgt-runtime-tab.png](img/banking-domain-usecase/acc-mgt-runtime-tab.png)
```

Comment:
This image does not exit.

Reply:
oops, ,missed committing that file. was in untracked files for some reason